integration-tests/python-enclave/host/src/test/kotlin/com/r3/conclave/integrationtests/python/enclave/PythonEnclaveTest.kt

```diff
@@ -39,21 +45,31 @@ class PythonEnclaveTest {
     }
 
     @Test
-    fun receive_from_untrusted_host() {
-        val data = "hello world".toByteArray()
-        val signature = enclaveHost.callEnclave(data)!!
-        with(enclaveHost.enclaveInstanceInfo.verifier()) {
-            update(data)
-            assertThat(verify(signature)).isTrue
-        }
-    }
-
-    @Test
-    fun receive_enclave_mail() {
+    fun run_mnist_rnn_pytorch_example() {
```

Comment:
We're no longer in Python! Use backticks and full sentence for the test method anem